client/my-sites/site-settings/media-settings.jsx

```diff
@@ -29,10 +31,16 @@ import {
 	isJetpackModuleUnavailableInDevelopmentMode,
 	isJetpackSiteInDevelopmentMode
 } from 'state/selectors';
+import { getMediaStorage } from 'state/sites/media-storage/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { getSitePlanSlug } from 'state/sites/selectors';
+import {
+	getSitePlanSlug,
+	getSiteSlug,
+} from 'state/sites/selectors';
 import { updateSettings } from 'state/jetpack/settings/actions';
+import QueryMediaStorage from 'components/data/query-media-storage';
 import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import PlanStorageBar from 'blocks/plan-storage/bar';
```

Comment:
I've spent some time going down this road and I'm feeling like it's rather costly and has limited benefit. Feeling like it's a case of YAGNI.

* There seems to be low opportunity for reuse of the block.
* It doesn't save any logic or connected state in this component. We'll still need to connect all the same state to know whether to render the bar or the message.
* Conditionally rendering the bar _or_ message could go into a single block, but I think there's even _less_ opportunity to reuse that because it's tightly tied to this UI.
* Both `VideoStorageBar` and `VideoStorageMessage` blocks could be implemented and rendered, they'd be smart enough to know which renders null and which doesn't, but it feels like overkill as I see one or both of the blocks as _only_ being useful here, in this component.

See #16806 for the VideoStorage (bar) block implementation.

What do you think?